src/Agent.Plugins/GitCliManager.cs

```diff
@@ -26,15 +26,15 @@ private static Encoding _encoding
                 : null;
         }
 
-        private readonly Dictionary<string, string> gitEnv = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
+        protected readonly Dictionary<string, string> gitEnv = new Dictionary<string, string>(StringComparer.OrdinalIgnoreCase)
```

Comment:
How do we do naming for `protected` elsewhere? Is it camel or pascal case? Let's follow whatever we do elsewhere. My gut says Pascal case is the right way to go.

Reply:
Its inconsistent in our library, I'm going to leave it as is since we already have some public methods with conflicting names (e.g. `public virtual async Task<Version> GitVersion(AgentTaskPluginExecutionContext context)`)